app/src/main/kotlin/org/koitharu/kotatsu/core/os/AppShortcutManager.kt

```diff
@@ -86,47 +81,37 @@ class AppShortcutManager @Inject constructor(
 	}
 
 	fun isDynamicShortcutsAvailable(): Boolean {
```

Comment:
I see that `ShortcutManagerCompat` do some stuff even if SDK_INT < N_MR1, so this check might be omitted

Reply:
It allows dynamic shortcuts to be shown under Direct Share on older versions of Android: https://medium.com/google-developer-experts/exploring-android-q-sharing-shortcuts-a59e3b4821da